style.css

```diff
@@ -321,6 +324,8 @@ textarea:hover {
   flex: none;
   order: 0;
   flex-grow: 0;
+  float: left;
+  padding: 4px;
 }
 
 .btn1 {
```

Comment:
- [ ] Kindly use button interactions such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in your chosen template :+1: